### ISSUE TITLE
[BUG] golang.stackrox.io/kube-linter latest version is v0.4.0

### ISSUE THREAD

**Issue by czeslavo** [stackrox/kube-linter#495]
**Describe the bug**
When go-getting the `golang.stackrox.io/kube-linter` module, the latest version is v0.4.0 while the GitHub repository already has a [v0.6.0](https://github.com/stackrox/kube-linter/releases/tag/0.6.0) tag. 

**To Reproduce**
```shell 
$ go list -m -versions golang.stackrox.io/kube-linter
golang.stackrox.io/kube-linter v0.4.0
```

**Expected behavior**
The newest versions should be published to `golang.stackrox.io/kube-linter` so users can `go get` them properly.

**Comment by czeslavo** [comment]
Duplicate of https://github.com/stackrox/kube-linter/issues/494
